### DISCUSSION TITLE
Import-DbaCsv : Convert Exponential Notation to Decimal numbers

### DISCUSSION THREAD

**Discussion by kappamega** [dataplat/dbatools#8507] [General] [answered]
If we have occasional  Exponential Notation value in CSV file - any suggestion how to handle it. 

Example:
"HAT110_3",22.8407574420657
"HAT110_3",29.9061006689284
"HAT110_3",47.1852814503376
"HAT110_3",7.40740192073375e-05
"HAT110_3",47.1852814503376

-- Exponential Values generates following error
Input string was not in a correct format.
At C:\Program Files\WindowsPowerShell\Modules\dbatools\1.1.127\allcommands.ps1:148329 char:9
+         throw $records[0]
+         ~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], Exception
    + FullyQualifiedErrorId : dbatools_Import-DbaCsv

**Comment by potatoqualitee** [comment, marked as answer]
does that happen even when your data type is nvarchar(max)?

**Reply by kappamega** [reply]
Works Perfect when converted to nvarchar(max)

**Reply by potatoqualitee** [reply]
sweet, k that's just a datatype problem then? if it breaks when its the proper datatype, then perhaps import to an intermediate table then do some conversions to get it into the final table
